src/Factories/FunFunFactory.php

```diff
@@ -466,6 +466,10 @@ public function getSkinTwig(): Environment {
 				$packageFactory->newAssetPackages()
 			);
 			$locale = Locale::parseLocale( $this->getLocale() );
+			if ( $locale === null ) {
+				$locale = [ 'language' => 'de' ];
+			}
+
```

Comment:
You could DRY this code up a bit by putting the lookup into a `getLanguage` function that encapsulates the check and replace the `$locale['language']` access with `$this->getLanguageFromLocale()`.

Inside the new method you could also use  `Locale::getPrimayLanguage()` instead of `Locale::parseLocale()`, see https://www.php.net/manual/en/locale.getprimarylanguage.php

Reply:
I did the changes. I hope that is what you meant.